iphone/CoreApi/CoreApi/Bookmarks/MWMBookmarksManager.h

```diff
@@ -24,6 +24,7 @@ typedef void (^PingCompletionBlock)(BOOL success);
 typedef void (^ElevationPointChangedBlock)(double distance);
 typedef void (^SearchBookmarksCompletionBlock)(NSArray<MWMBookmark *> *bookmarks);
 typedef void (^SortBookmarksCompletionBlock)(NSArray<MWMBookmarksSection *> * _Nullable sortedSections);
+typedef void (^SharingResultCompletionHandler)(MWMBookmarksShareStatus status, NSURL * _Nullable url);
```

Comment:
1. Can the URL be null? Should it be documented here or somewhere else?
2. Is it a URL to a local file? It would be great to document it/rename it (urlToALocalFile?)

Reply:
1. Yes. In case of `status != .success` URL will be null
2. Yes it is URL to the local temporary file

Reply:
@biodranik =i can add some documentation to clarify the usage of these methods:
```objc

/**
 Shares a specific category with the given group ID.

 @param groupId The identifier for the category to be shared.
 @param completion A block that handles the result of the share operation. This block has no return value and takes two parameters:
                   - status: The status of the share operation, of type `MWMBookmarksShareStatus`.
                   - urlToALocalFile: The local file URL containing the shared data. This parameter is guaranteed to be non-nil only if `status` is `MWMBookmarksShareStatusSuccess`. In other cases, it will be nil.
*/
- (void)shareCategory:(MWMMarkGroupID)groupId completion:(SharingResultCompletionHandler)completion;

/**
 Shares all categories.

 @param completion A block that handles the result of the share operation. This block has no return value and takes two parameters:
                   - status: The status of the share operation, of type `MWMBookmarksShareStatus`.
                   - urlToALocalFile: The local file URL containing the shared data. This parameter is guaranteed to be non-nil only if `status` is `MWMBookmarksShareStatusSuccess`.  In other cases, it will be nil.
*/
- (void)shareAllCategoriesWithCompletion:(SharingResultCompletionHandler)completion;
```

<img width="380" alt="image" src="https://github.com/organicmaps/organicmaps/assets/79797627/f464d7d7-8197-4c3a-9fdf-e0b5716d94ed">